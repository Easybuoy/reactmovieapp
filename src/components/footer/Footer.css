.me{
    display: flex;
    justify-content: flex-end;
}