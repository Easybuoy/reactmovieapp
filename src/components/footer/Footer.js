import React from 'react';
import { Col, Container, Row, Footer, Animation } from 'mdbreact';
import { Fa } from 'mdbreact';
import tmdbimg from './tmdb.png';

class FooterPage extends React.Component {
    render(){ 
        return(
            <Footer color="blue" className="font-small pt-4 mt-4">
                <Container fluid className="text-center text-md-left">
                    <Row>
                    <Col sm="4">
                        <a style={{'color': 'white'  }} target="_blank" href= "https://github.com/Easybuoy"><Fa icon="github" size="2x" className="fav" /></a>
                        <a style={{'color': 'white'}} target="_blank" href= "https://www.freecodecamp.org/easybuoy"><Fa icon="free-code-camp" size="2x" className="fav" /></a>
                        <a style={{'color': 'white'}} target="_blank" href= "https://codepen.io/easybuoy-the-flexboxer/"><Fa icon="codepen" size="2x" className="fav" /></a>
                        <a style={{'color': 'white'}} target="_blank" href= "https://www.linkedin.com/in/easybuoy/"><Fa icon="linkedin" size="2x" className="fav" /></a>
                        <a style={{'color': 'white'}} target="_blank" href= "https://stackoverflow.com/users/story/8326940"><Fa icon="stack-overflow" size="2x" className="fav" /></a>
                        <a style={{'color': 'white'}} target="_blank" href= "http://twitter.com/easybuoy/"><Fa icon="twitter" size="2x" className="fav" /></a>


                    </Col>
                    <Col sm="4">
                        
                        
                    </Col>
                    <Col sm="4">
                    <Animation type="slideInRight">
                    <img className="d-block w-100" src={tmdbimg} alt="Powered By The Movie Db" />
                    {/* style={{height: '100px', width: '100px'}} */}
                    </Animation>
                    </Col>
                    </Row>
                </Container>
                <div className="footer-copyright text-center py-3">
                    <Container fluid>
                        &copy; {(new Date().getFullYear())} Copyright: <a href="#"> Movie App </a>
                    </Container>
                </div>
            </Footer>
        );
    }
}

export default FooterPage;