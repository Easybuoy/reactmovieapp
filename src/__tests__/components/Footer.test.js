/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/footer/Footer';

describe('<Footer />', () => {

  it('renders the Footer component correctly', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
});
})

