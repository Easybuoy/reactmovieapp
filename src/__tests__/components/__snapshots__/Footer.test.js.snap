// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders the Footer component correctly 1`] = `ShallowWrapper {}`;
